CHANGELOG.md

```diff
@@ -8,15 +8,50 @@ vNext
 -
 -
 -
-- Added lifted conditional `nn.cond`
-- Handle rate==1.0 edgecase in Dropout.
 -
 -
 -
 -
 -
 -
 -
+-
+-
+-
+-
+-
+-
+-
+-
+-
+-
+
+0.4.2
+-----
+
+New features:
```

Comment:
so there are no breaking changes?

I find it useful to have short readable changelog, not necessarily with all the PRs and who wrote them (I can easily get that view in Github from https://github.com/google/flax/compare/v0.4.0...v0.4.1 ), but the most important ones, possibly grouped by "new feature" & "breaking changes" etc.

But that's just my preference, YMMV.